go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/pions/webrtc
 
 require (
-	github.com/pions/datachannel v1.2.0
+	github.com/pions/datachannel v1.2.1-0.20190212000153-79846f5a6cf6
```

Comment:
Maybe do a release of the dependencies once your PRs land.

Reply:
Yes. I will leave a note for it in #334